.github/workflows/performance.yml

```
@@ -0,0 +1,185 @@
+name: Performance
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

```suggestion
concurrency:
  group: ${{ github.workflow }}-${{ github.event.pull_request.number ||
github.sha }}
  cancel-in-progress: true
```

I was only recently pointed out the downside of the default GitHub suggests, that also cancels in-progress tests in the default branch, which may not be something you want to have "tests always pass" in the default branch. Found in https://mail.python.org/archives/list/pypa-committers@python.org/thread/PCBCQMJF64JGRBOX7E2EE4YLKHT4DI55/